.github/workflows/icu_ci.yml

```diff
@@ -58,6 +83,62 @@ jobs:
           CFLAGS: -Wimplicit-fallthrough
           CXXFLAGS: -Wimplicit-fallthrough
 
+  # Out of source build with gcc 8 and c++14; executes icuinfo.
+  gcc-8-stdlib_14:
```

Comment:
nit: It seems strange to see a YAML key with an underscore. On the other hand, down below in this PR, `INTLTEST_OPTS` is a key that must match the shell env variable to be created.

Reply:
Changed to 'stdlib14'.